CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+# v0.4.8
+
+## Description
+Security release to upgrade the Docker image to the latest version of Alpine linux that fixes
+this bug: https://justi.cz/security/2018/09/13/alpine-apk-rce.html
+
+Diff: https://github.com/runatlantis/atlantis/compare/v0.4.7...v0.4.8
```

Comment:
@majormoses how's this! 😃

Reply:
that works, I actually like putting that link in the header but its still an improvement nonetheless.